test/WhereClauseTest.php

```diff
@@ -1,22 +1,23 @@
 <?php
 
-use ActiveRecord\ConnectionManager;
-use ActiveRecord\DatabaseException;
 use ActiveRecord\Exception\ExpressionsException;
 use ActiveRecord\WhereClause;
-use PHPUnit\Framework\TestCase;
 
-class WhereClauseTest extends TestCase
+class WhereClauseTest extends DatabaseTestCase
 {
-    protected $connection;
-
-    public function setUp(): void
-    {
-        parent::setUp(); // TODO: Change the autogenerated stub
-        try {
-            $this->connection = ConnectionManager::get_connection();
-        } catch (DatabaseException $e) {
-            $this->markTestSkipped('failed to connect. ' . $e->getMessage());
+    private string $id;
+    private string $name;
+
+    public function setUp(string $connection_name = null): void
+    {
+        if (parent::connect('pgsql')) {
```

Comment:
Why is this still hard-coded to `pgsql`?

Reply:
To test Postgres if it's available, fall back to MySQL if not. Both pass regression.

Reply:
In that case let's just put it back the way it was. I'm tinkering around with running the entire test suite across all connection types in https://github.com/php-activerecord/activerecord/pull/88

Reply:
Removed setUp function.